src/lib/viewers/BaseViewer.js

```diff
@@ -108,6 +109,12 @@ class BaseViewer extends EventEmitter {
         this.options = options;
         this.cache = options.cache;
         this.previewUI = options.ui;
+
+        const fileOptions = options.fileOptions || {};
```

Comment:
const { fileOptions } = options;

You already default fileOptions to {} in Preview.js